inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/constant/TaskConstants.java

```diff
@@ -117,6 +117,14 @@ public class TaskConstants extends CommonConstants {
     public static final String JOB_KAFKA_READ_TIMEOUT = "job.kafkaJob.read.timeout";
     public static final String TASK_KAFKA_AUTO_COMMIT_OFFSET_RESET = "task.kafkaJob.autoOffsetReset";
 
+    // Pulsar task
+    public static final String TASK_PULSAR_TENANT = "task.pulsarTask.tenant";
+    public static final String TASK_PULSAR_NAMESPACE = "task.pulsarTask.namespace";
+    public static final String TASK_PULSAR_TOPIC = "task.pulsarTask.topic";
+    public static final String TASK_PULSAR_SERVICE_URL = "task.pulsarTask.serviceUrl";
+    public static final String TASK_PULSAR_SCAN_STARTUP_MODE = "task.pulsarTask.scanStartupMode";
+    public static final String TASK_PULSAR_RESET_TIME = "task.pulsarTask.resetTime";
+
```

Comment:
Currently, we have `Binlog job` and `Kafka job`,  is it better for us to add new type called `Pulsar job` instead of `Pulsar task`?